### DISCUSSION TITLE
is this Production ready ?

### DISCUSSION THREAD

**Discussion by sobhardwaj** [mantinedev/mantine#165] [Q&A] [answered]
Hi, I would like to ask how many using this in production ? if any issue please highlight that as well ?

**Comment by rtivital** [comment, marked as answer]
Hi, best way to see how Mantine works in production is to browse the docs website – it is always guaranteed to use all components and hooks, you can check each component behavior in different browsers (including mobile devices). Apart from demos, docs website also uses Mantine to build all other more complex pages and components (home page, navigation, etc.).

I use Mantine at work for 3 applications – Internal CMS, Education platform and integrated chat service (cannot show it though as all ot it is under authentication) and never had any problems.
